src/orion/algo/dehb/logger.py

```diff
@@ -0,0 +1,27 @@
+"""Utility to silence loguru"""
+
+
+def passmethod(*_args, **_kwargs):
+    """Method that does nothing"""
+    pass
+
+
+# pylint:disable=too-few-public-methods
+class NoLogger:
+    """Fake logger to prevent file creation by loguru"""
+
+    def __getattr__(self, _item):
+        """always return something that does nothing"""
+        return passmethod
+
+
+def remove_loguru():
+    """Override loguru logger class to ignore all its call"""
+    # TODO: wtf is that?
```

Comment:
Nit: remove

Reply:
Removed in https://github.com/Epistimio/orion/commit/ebfff61c98c062202355f2ce3c308661e9fc80a1